javascript/step_2.js

```diff
@@ -12,5 +12,4 @@ app.use(bodyParser.urlencoded({
 app.use(bodyParser.json());
 
 // Listen
-
```

Comment:
I don't see any difference between step_2 and the solution one, apart from the comment and a missing new line... I'm confused

Reply:
that's correct, there's nothing to do in step 2